romeo/src/stacks_client.rs

```diff
@@ -76,7 +76,11 @@ impl StacksClient {
 	{
 		let request_url = request_builder().url().to_string();
 
-		let res = retry(|| self.http_client.execute(request_builder())).await?;
+		let res = retry(|| {
+			self.http_client
+				.execute(self.add_stacks_api_key(request_builder()))
+		})
+		.await?;
```

Comment:
Does this remove a warning? Why edit this?

Reply:
> Does this remove a warning? Why edit this?

There is the `add api key`